source/documentation/10-switching-to-production.md

```diff
@@ -28,11 +46,10 @@ If you want to use 3D Secure authentication for your payments, ask your Worldpay
 
 To set up ePDQ to work with GOV.UK Pay, you must log into the ePDQ admin site and:
 
-1. Add payment methods to your account.
-1. Set up account security parameters.
-1. Set up notification settings.
-1. Set up an API user.
-1. Set up GOV.UK account credentials.
+1. [Add payment methods to your account](https://govukpay-docs.cloudapps.digital/#add-payment-methods-to-your-account).
+1. [Set up account security parameters](https://govukpay-docs.cloudapps.digital/#set-up-account-security-parameters).
+1. [Set up notification settings](https://govukpay-docs.cloudapps.digital/#set-up-notification-settings).
+1. [Set up an API user](https://govukpay-docs.cloudapps.digital/#set-up-an-api-user).
```

Comment:
is there a reason why you list these links and then go through the content contained in those links in detail underneath. Feels a bit confusing and different from what we do for worldpay?

Reply:
I added the links in so that users could jump to the section they want to refer to if necessary. However, happy to remove if you think it is confusing.

Reply:
I found it a bit confusing, but it might just be me. Maybe just the fact that it wasn't consistent with worldpay section